generator/sbpg/targets/resources/sbp_construct_template.py.j2

```diff
@@ -29,7 +31,7 @@ from (((module_path))).(((i))) import *
 ((* for m in msgs *))
   ((*- if m.static *))
     ((*- if not m.sbp_id *))
-class ((( m.identifier )))(object):
+class ((( m.identifier )))():
```

Comment:
https://docs.python.org/2/reference/datamodel.html#slots

In new style objects __dict__ is not created if __slots__ is created. I needed the __dict__ for size attribute but I agree this is not the best approach. I have reworked this so that class is again new style and size is returned explicitly from from_binary().